src/ControlSystem/NamespaceDocs.hpp

```diff
@@ -6,3 +6,9 @@
 /// \ingroup ControlSystemGroup
 /// Control systems and related functionality
 namespace control_system {}
+
+namespace control_system {
```

Comment:
You can delete these lines and the closing brace in line 8. Or just merge as is.

Reply:
Yeah this is better. I'll probably be adding a couple more namespaces in the future so this makes it easier to add.